### PR TITLE
Add run jest file with coverage

### LINKED ISSUE #58: Run with coverage as a separate command
I know that we can specify the --coverage in the run options, but in my usual workflow, I write/run a bunch of tests one by one for a file, and when I'm done, I want to run the whole file with coverage to see if I missed something. So adding the --coverage in the options runs it every time, which slows things down so it's a no go.

If I had a "Run with coverage" command, I could set a separate keybinding for it and run it easily after I'm done with a test file. The workaround for now is to run the test for the whole file, and then edit the command in the terminal to add --coverage.

### PR DESCRIPTION
**Pull request by EstefaniaGuardado**
Closes #58

### CODE REVIEW

File: src/jestRunner.ts [hunk 1]
```diff
@@ -58,6 +58,25 @@ export class JestRunner {
     await this.runTerminalCommand(command);
   }
 
+  public async runCurrentFileWithCoverage() {
```
Comment (viciteufel) [inline src/jestRunner.ts:61]: I would suggest to add an `options` parameter to the `runCurrentFile()` method, because there is some duplicated code now.

Comment (EstefaniaGuardado) [reply src/jestRunner.ts]: I applied the feedback @viciteufel, thank you very much!

File: src/jestRunner.ts [hunk 2]
```diff
@@ -50,7 +50,14 @@ export class JestRunner {
...
     const filePath = editor.document.fileName;
-    const command = this.buildJestCommand(filePath);
+    let command: string;
+
+    if (options) {
+      const testName = undefined;
+      command = this.buildJestCommand(filePath, testName, options);
+    } else {
+      command = this.buildJestCommand(filePath);
+    }
```
Comment (viciteufel) [inline src/jestRunner.ts:60]: Thanks for updating the pull request! I have another small change request:
Why the `if` statement? `options` is also an optional parameter in the `buildJestCommand()` function, so those 8 lines could be rewritten to

```
const command = this.buildJestCommand(filePath, undefined, options);
```

Comment (EstefaniaGuardado) [reply src/jestRunner.ts]: Thanks for the feedback! I changed the code.

**Comment by viciteufel** [comment]
Thanks for your contribution. Your changes are available in version 0.4.18

**Pull request merged**